test/TestServerProjects/body-date/Generated/DateClient.cs

```diff
@@ -176,7 +176,7 @@ public virtual Response<DateTimeOffset> GetUnderflowDate(CancellationToken cance
         /// <summary> Put max date value 9999-12-31. </summary>
         /// <param name="dateBody"> date body. </param>
         /// <param name="cancellationToken"> The cancellation token to use. </param>
-        public virtual async Task<Response> PutMaxDateAsync(DateTimeOffset dateBody, CancellationToken cancellationToken = default)
+        public virtual async Task<Response> PutMaxDateAsync(string dateBody, CancellationToken cancellationToken = default)
```

Comment:
Why did `DateTimeOffset` change to string?

Reply:
Some of the type changes are caused by a bug in modelerfour and I updated its version to use the one in this fix [PR](https://github.com/Azure/autorest/pull/4477). They are fixed now.